File: app/src/main/java/knf/kuma/tv/exoplayer/TVPlayer.java
package knf.kuma.tv.exoplayer;

import android.os.Bundle;
import android.view.WindowManager;

import androidx.annotation.Nullable;
import knf.kuma.tv.TVBaseActivity;

public class TVPlayer extends TVBaseActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        getWindow().addFlags(WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON);
        addFragment(PlaybackFragment.get(getIntent().getExtras()));
    }
}

File: app/src/main/java/knf/kuma/tv/exoplayer/TVPlayer.kt
package knf.kuma.tv.exoplayer

import android.os.Bundle
import android.view.WindowManager
import knf.kuma.tv.TVBaseActivity
import kotlinx.coroutines.ExperimentalCoroutinesApi
import kotlin.contracts.ExperimentalContracts

@ExperimentalCoroutinesApi
@ExperimentalContracts
class TVPlayer : TVBaseActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        window.addFlags(WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON)
        addFragment(PlaybackFragment[intent.extras])
    }
}
